invoicing/functions/server/slow.js

```diff
@@ -31,5 +32,47 @@ exports.REST = {
                 return resolve({data: {success: true, time}});
             }, time * 1000);
         });
+    },
+
+    $slower: {
+        module: "invoicing",
+        name: "slower",
+        scope: "public",
+        description: "waits some seconds before responding",
+        restricted: false,
+        args: [
+            {
+                name: "time",
+                type: "number",
+                description: "number of seconds to wait; defaults to 60"
+            }
+        ],
+        content: [
+            {
+                name: "multi",
+                type: "number",
+                description: "multiplier; defaults to 1"
+            }
+        ],
+        output: "{success: true, time: 'seconds waited'}",
+        example: "/invoicing/slow/slow?time=120  =>  {success: true, time: 120}",
+    },
+    slower(args, content, serv, role, session) {
```

Comment:
Could be better to find a example related to the sample, we can discuss about that.